src/menu-private.h

```diff
@@ -147,8 +148,9 @@ gboolean brisk_menu_window_setup_session(BriskMenuWindow *self);
 
 /* Settings */
 void brisk_menu_window_init_settings(BriskMenuWindow *self);
+void brisk_menu_applet_init_settings(BriskMenuWindow *self, BriskMenuApplet *applet);
 void brisk_menu_window_update_search(BriskMenuWindow *self, SearchPosition pos);
-void brisk_menu_window_pump_settings(BriskMenuWindow *self);
+void brisk_menu_window_pump_settings(BriskMenuWindow *self, BriskMenuApplet *applet);
```

Comment:
Feels perhaps a bit spaghetti that the applet references the window to talk to the key binder to the window to the applet ..